spec/components/status_card/view_preview.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module StatusCard
+  class ViewPreview < ViewComponent::Preview
+    def single_card
+      render inline: "<div class=\"govuk-grid-row govuk-!-margin-bottom-6\"><%= render(StatusCard::View.new(card))%></div>", locals: { card: card }
```

Comment:
Is there a reason we can't do `render(StatusCard::View.new(card))`?

Reply:
Thanks, fixed